frontend/src/concepts/pipelines/content/pipelinesDetails/taskDetails/TaskDetailsCodeBlock.tsx

```diff
@@ -15,7 +15,7 @@ const TaskDetailsCodeBlock: React.FC<TaskDetailsCodeBlockProps> = ({ id, content
   const [copied, setCopied] = React.useState(false);
 
   const clipboardCopyFunc = (text: string) => {
-    navigator.clipboard.writeText(text.toString());
+    navigator.clipboard.writeText(String(text));
```

Comment:
This is already of type `string`.
```suggestion
    navigator.clipboard.writeText(text);
```